2.illumination_correction/scripts/0.cp_whole_image_qc.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Run whole image QC pipeline in CellProfiler
+# 
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pathlib
+import pprint
+
+import sys
+
+sys.path.append("../utils")
```

Comment:
Hmm, not sure! I really want to make CP parallel as it's own package since I have to copy and paste it into every project. I think I will save this and I want to discuss with you what it would take to make the simple python function of cp parallel as a package.